docs/channel-slack-setup.md

```diff
@@ -12,25 +12,51 @@ Before going further, we assume that you already have:
 - a Slack Account
 - a [Slack Workspace](https://slack.com/create#email)
 
-### Slack App and Bot User
+### Create a Slack App
 
 Create a [Slack App](https://api.slack.com/apps?new_app=1) if you haven't.
 
- <p><img width="800" src="https://user-images.githubusercontent.com/662387/71443858-64c54580-2748-11ea-9d64-7fb321a176b0.png"></p>
- <p><img width="800" src="https://user-images.githubusercontent.com/662387/71443859-655ddc00-2748-11ea-942d-22d4378b8a28.png"></p>
+ <p><img width="800" src="https://user-images.githubusercontent.com/563929/82567644-3a20d100-9bb0-11ea-9c53-a04b62340db0.png"></p>
 
-Create a bot user within your Slack App.
+Click the `Create New App` button.
 
-<p><img width="800" src="https://user-images.githubusercontent.com/662387/71443860-655ddc00-2748-11ea-98ce-d54f96b2ea9f.png"></p>
+ <p><img width="800" src="https://user-images.githubusercontent.com/563929/82567788-72c0aa80-9bb0-11ea-8233-48b604d3495e.png"></p>
 
-<p><img width="800" src="https://user-images.githubusercontent.com/662387/71443861-655ddc00-2748-11ea-805d-31326486c049.png"></p>
+Fill the `App Name` and `Development Slack Workspace` and then click the `Create App` button.
 
-Remember to install the Slack App in your workspace.
+We recommand creating a new workspace for experiment.
 
-<p><img width="800" src="https://user-images.githubusercontent.com/662387/71443862-65f67280-2748-11ea-83a8-ca04340d6217.png"></p>
-<p><img width="800" src="https://user-images.githubusercontent.com/662387/71443863-65f67280-2748-11ea-8de7-2be1f0419729.png"></p>
+### Install Slack App to Workspace
 
-> **Note:** If you are not familiar with how Slack bots work, you can find detailed instructions from Dialogflow's [Slack Integration Document](https://cloud.google.com/dialogflow/docs/integrations/slack)
+You can find the `install App to Workspace` button in [Slack Developer Console](https://api.slack.com/apps) → \${YourApp} → OAuth & Permissions
+
+ <p><img width="800" src="https://user-images.githubusercontent.com/563929/82569943-80c3fa80-9bb3-11ea-997c-346a685d1adf.png"></p>
+
+The `install App to Workspace` button is disabled, to enable the button, you need to setup at least one of the permissions the Slack app need.
```

Comment:
install App to Workspace -> Install App to Workspace

Reply:
.... is disabled. To enable the button, you need to set up at least one of the permissions the Slack app need.